components/common/ConnectWallet/WalletAssetIdentity.vue

```diff
@@ -1,16 +1,32 @@
 <template>
   <div class="flex items-center justify-between">
-    <IdentityItem
-      :account="account"
-      :label="display || shortenedAddress"
-      :prefix="urlPrefix"
-      variant="button" />
-    <div class="flex flex-col">
+    <NuxtLink
+      :to="`/${urlPrefix}/u/${account}`"
+      class="w-full"
+      @click="closeModal">
```

Comment:
why wrap it in `NuxtLink` as `IdentityItem` is already doing likewise

Reply:
green section needs to be clickable. 

![CleanShot 2024-01-24 at 13 41 48@2x](https://github.com/kodadot/nft-gallery/assets/44554284/b7d19873-2214-4876-ae2c-360793bd775e)

Reply:
Alright design has changed